### PR TITLE
Fix punctuation and "Share"

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: user_manual/files/sharing.rst [hunk 1]
```diff
@@ -27,16 +27,16 @@ A number of options are available for public *folder* shares:
 .. figure:: ../images/sharing_public_folder.png
 
 * **Read only** to allow viewing and downloading
-* ** Allow upload and editing**
-* With ** File drop**, the sharee can only upload files to a folder without seeing the files that are already in that folder.
```
Comment (tflidd) [inline user_manual/files/sharing.rst:31]: that is a person to which the folder is shared (https://en.wiktionary.org/wiki/sharee)
If that term is perhaps less know there might be a better word or some explanation when hovering over the term...

File: user_manual/files/sharing.rst [hunk 2]
```diff
@@ -48,12 +48,12 @@ When sharing with users, groups, circles or members of a Talk conversation, righ
 
 .. figure:: ../images/sharing_internal.png
 
-As a sharee, you can configure if you automatically want to accept all incoming shares and have them added to your root folder, or if you
```
Comment (tflidd) [inline user_manual/files/sharing.rst:51]: same, that's a person...

**Review by tflidd — CHANGES_REQUESTED** [review]
Apart from the `sharee`, the rest looks ok for me

File: user_manual/files/sharing.rst (hunk 1 above)
Comment (Valdnet) [reply user_manual/files/sharing.rst:31]: I understand that word now. Well, we leave them unchanged.

File: user_manual/files/sharing.rst (hunk 2 above)
Comment (Valdnet) [reply user_manual/files/sharing.rst]: @tflidd Also corrected :wink:.

**Review by tflidd — APPROVED** [review]

**Pull request merged**